database/sql/V075__add_call_data_to_quotes_and_order_quotes.sql

```diff
@@ -0,0 +1,8 @@
+-- Step 1: Add a new column to the quotes table
+ALTER TABLE quotes 
+    ADD COLUMN call_data bytea;
```

Comment:
Why don't we want a `verified` flag here as well?

Reply:
Added.